packages/@okta/vuepress-site/docs/guides/okta-account-management-policy/main/index.md

```diff
@@ -0,0 +1,396 @@
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'Authorization: SSWS {apiToken}' \
+--data '{
+    "system": false,
+    "type": "ACCESS_POLICY",
+    "id": "rul2p9xACRTDf9qZV0g4",
+    "name": "Test rule",
+    "priority": 1,
+    "status": "ACTIVE",
+    "created": "2024-07-16T20:27:25.000Z",
+    "lastUpdated": "2024-07-16T20:27:25.000Z",
+    "conditions": {
+        "userType": {
+            "include": [],
+            "exclude": []
+        },
+        "network": {
+          "connection": "ZONE",
+          "include": [
+            "nzo5dtwhzly9mXvlN0g7"
+            ]
+        },
+        "people": {
+            "users": {
+                "exclude": [],
+                "include": []
+            },
+            "groups": {
+                "include": [],
+                "exclude": []
+            }
+        },
+        "device": {
+            "registered": true,
+            "managed": true
+        },
+        "riskScore": {
+            "level": "LOW"
+        },
+        "elCondition": {
+            "condition": "accessRequest.operation == '\''enroll'\'' && ( accessRequest.authenticator.key == '\''okta_verify'\'' || accessRequest.authenticator.key == '\''webauthn'\'' || accessRequest.authenticator.key == '\''smart_card_idp'\'' || accessRequest.authenticator.key == '\''yubikey_token'\'' )"
+        },
+        "platform": {
+            "include": []
+        }
+    },
+    "actions": {
+        "appSignOn": {
+            "access": "ALLOW",
+            "verificationMethod": {
+                "factorMode": "2FA",
+                "reauthenticateIn": "PT0S",
+                "constraints": [
+                    {
+                        "possession": {
+                            "userPresence": "OPTIONAL"
+                        }
+                    }
+                ],
+                "type": "ASSURANCE"
+            }
+        }
+    },
+    "initialMode": false,
+    "_links": {
+        "self": {
+            "href": "https://brent-stellar-org.trexcloud.com/api/v1/policies/rst1k0mWSjjvYzqQv0g4/rules/rul2p9xACRTDf9qZV0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "deactivate": {
+            "href": "https://brent-stellar-org.trexcloud.com/api/v1/policies/rst1k0mWSjjvYzqQv0g4/rules/rul2p9xACRTDf9qZV0g4/lifecycle/deactivate",
+            "hints": {
+                "allow": [
+                    "POST"
+                ]
+            }
+        }
+    }
+}'
+```
+
+### User experience
+
+Users must be on a managed device, inside a trusted network zone, and demonstrate low risk behavior before they enroll the designated phishing-resistant authenticator. If they don't meet all of these requirements, they're denied enrollment. That means that they also can't access any apps with phishing-resistant authentication policies.
```

Comment:
```suggestion
Users must be on a managed device, inside a trusted network zone, and demonstrate low risk behavior before they enroll the designated phishing-resistant authenticator. If they don't meet these requirements, all fields in their profile settings are read-only, including the Reset, Update, and Remove options for their existing security methods. The phishing-resistant authenticators that they haven't enrolled are hidden, which means that they can't access any apps with phishing-resistant authentication policies.
```

Reply:
this is one the recent changes I made based on Jake's feedback (may not be necessary for your purposes but it was something the team discussed including as a UX change)

Reply:
Sounds good. Thank you!